pkgs/top-level/aliases.nix

```
@@ -730,6 +730,9 @@ mapAliases ({
   youtubeDL = youtube-dl;  # added 2014-10-26
   ytop = throw "ytop has been abandoned by upstream. Consider switching to bottom instead";
   zdfmediathk = mediathekview; # added 2019-01-19
+  gnomeExtensions = self // {
+    unite-shell = self.unite; # added 2021-01-19
+  };
```

pretty sure self refers to pkgs
```suggestion
  gnomeExtensions = gnomeExtensions // {
    unite-shell = gnomeExtensions.unite; # added 2021-01-19
  };
```